MantleTests/MTLTestModel.h

```diff
@@ -97,6 +97,20 @@ extern const NSInteger MTLTestModelNameMissing;
 @property (readonly, nonatomic, weak) id weakProperty;
 @property (readonly, nonatomic, strong) id strongProperty;
 
+@property (readonly, nonatomic, copy) NSDate *dateFromProtocol;
+
+@end
+
+@protocol MTLDateProtocol <NSObject>
+
+@property (readonly, nonatomic, copy) NSDate *dateFromProtocol;
```

Comment:
Is `copy` relevant here? If not, can we make it strong?

Since the type of the property shouldn't matter, can we make this `id declaredInProtocol` or something like that?